flatpak/mypaint-Makefile

```diff
@@ -0,0 +1,4 @@
+all:
```

Comment:
This is mostly cosmetics, but flatpak-buillder has `simple` buildsystem since 0.9, skipping any buildsystem assumptions. [Example usage](https://github.com/flathub/org.wesnoth.Wesnoth/blob/da5aa2cb6fbc3411319665457d4a8e52a389a3fb/org.wesnoth.Wesnoth.json#L32,L46).